events/schema.py

```diff
@@ -141,6 +145,18 @@ def resolve_enrolment_count(self, info, **kwargs):
     def resolve_capacity(self, info, **kwargs):
         return self.get_capacity()
 
+    def resolve_child_has_free_spot_notification_subscription(self, info, **kwargs):
+        child_id = get_node_id_from_global_id(kwargs["child_id"], "ChildNode")
+        try:
+            return (
+                Child.objects.user_can_view(info.context.user)
+                .get(pk=child_id)
+                .free_spot_notification_subscriptions.filter(occurrence_id=self.id)
+                .exists()
+            )
+        except Child.DoesNotExist as e:
+            raise ObjectDoesNotExistError(e)
```

Comment:
Since this is a query without data modification, I'd prefer return `None` instead of throwing error here

Reply:
Yeah good point!

Changed it to (hopefully) always return `None` when the ID doesn't match a child. It became a bit messy because of possible invalid IDs.